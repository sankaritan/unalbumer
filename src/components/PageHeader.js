import React from "react";

const PageHeader = () => (
    <div>
        <h4 id='app-title'>Unalbumer</h4>
        <hr></hr>
    </div>
)

export default PageHeader;