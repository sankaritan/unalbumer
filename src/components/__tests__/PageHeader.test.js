import React from "react";
import PageHeader from "../PageHeader";
import Adapter from "enzyme-adapter-react-16";
import Enzyme, { shallow } from 'enzyme';

Enzyme.configure({ adapter: new Adapter() });

describe("Page Header component", () => {
it("PageHeader renders correctly", () => {
  const tree = shallow(<PageHeader />).debug();
  expect(tree).toMatchSnapshot();
});
});
