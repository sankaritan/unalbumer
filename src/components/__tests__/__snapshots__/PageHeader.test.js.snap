// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page Header component PageHeader renders correctly 1`] = `
"<div>
  <h4 id=\\"app-title\\">
    Unalbumer
  </h4>
  <hr />
</div>"
`;
